info_db.py

```diff
@@ -0,0 +1,81 @@
+import logging
+import sqlite3
+from pathlib import Path
+from sqlite3 import Error
+
+DATABASE = Path("info_database.db")
+
+CREATE_TABLE_SQL = """
+CREATE TABLE IF NOT EXISTS bin_info (
+symbol TEXT,
+status TEXT,
+);
+"""
+INSERT_INTO_SQL = """
+INSERT INTO bin_info
+(symbol,status)
+VALUES(?,?);
+"""
+SELECT_FROM_SQL = """
+SELECT * FROM bin_info;
+"""
+
+
+def create_connection(db_file: Path):
+    """
+    Create db file
+    :param db_file: path to db file to create
+    :return:
+    """
+    try:
+        conn = sqlite3.connect(db_file)
+        logging.info("Connection created successfully !")
+        return conn
+    except Error as create_conn_err:
+        logging.error(create_conn_err)
```

Comment:
This logging is general and just throw the text of err but do not send any info where exactly it happend for programmer. Just update this one and other logging in exception blocks to understand where exactly errs were thrown